torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from torch.fx.graph_module import GraphModule
+
+
+def get_all_users_of(index: int, fx_module: GraphModule) -> List[int]:
+
```

Comment:
nit: can you remove this empty line between the function definition and and the docstring?

Reply:
Done